content/_includes/TNHardwareGuide.md

```diff
@@ -194,30 +197,30 @@ Consider pre-flighting every storage device before putting it into production, p
 
 Take time to create a pool before deploying the system. 
 Subject it to as close to a real-world workload as possible to reveal individual drive issues and help determine if an alternative pool layout is better suited to that workload.
-Be cautious of used drives as vendors may not be honest or informed about their age and health.
-Check the number of hours on all new drives using `smartctl(8)` to verify they are not recertified.
-A drive vendor could also zero the hours of a drive during recertification, masking its true age.
+Be cautious of used drives, as vendors may not be honest or informed about their age and health.
+Verify vendors have not recertified all new drives by checking the hours using `smartctl(8)`.
+A drive vendor could also zero the hours of a drive during recertification, masking its age.
```

Comment:
try to avoid possessives try drive age or the age, even age of the drive.
```suggestion
A drive vendor could also zero the hours of a drive during recertification, masking the drive age.
```